tests/e2e/Services/Functions/FunctionsCustomServerTest.php

```diff
@@ -2666,4 +2666,147 @@ public function testCreateFunctionWithResponseFormatHeader()
 
         $this->assertEquals(204, $response['headers']['status-code']);
     }
+
+    public function testFunctionLogging()
+    {
+        // Preparations: Create Function
+        $folder = 'node';
+        $code = realpath(__DIR__ . '/../../../resources/functions') . "/$folder/code.tar.gz";
+        $this->packageCode($folder);
```

Comment:
lets make use of an existing test to just add the logging attribute. adding a whole new test for this will keep adding overhead to the test times. functions tests already takes the longest